Makefile

```diff
@@ -692,6 +701,20 @@ generate-core-manifests: $(CONTROLLER_GEN) ## Generate manifests for the core pr
 		output:webhook:dir=./config/webhook \
 		webhook
 
+.PHONY: verify-generate-manifests
+verify-generate-manifests: generate-manifests ## Verify if generated manifests files e.g. CRD, RBAC etc. need to be updated
+	$(eval DIFF=$(shell git diff --raw -- '*.go' | wc -c))
+	if [[ $(DIFF) != 0 ]]; then \
+		echo "Detected out of date manifest files. please run 'make generate-manifests'"; \
+		exit 1;\
+	fi
+
+.PHONY: verify-auto-gen-files
+verify-auto-gen-files: 
```

Comment:
'auto-gen' is a little misleading. If they're automatically generated, why verify at all, it should be a solved problem? :D